### PR TITLE
convert url values for weblit map to lowercase and replace spaces wit…

### PR DESCRIPTION
**Pull request by ScottDowne**
…h dashes

### CODE REVIEW

File: pages/web-literacy/WebLiteracyPage.jsx [hunk 1]
```diff
@@ -17,11 +17,23 @@ var topicContent = weblitcontent.topics;
...
+  str = str || "";
+  return str.toLowerCase().replace(" ", "-");
+}
+
+function parseRoute(route) {
+  route = route || "";
+  return route.split("-").map(function(item) {
+    return item.charAt(0).toUpperCase() + item.substr(1).toLowerCase();
+  }).join(" ");
+}
```
Comment (ScottDowne) [inline pages/web-literacy/WebLiteracyPage.jsx]: I considered making these a module that we can require in, and I'm fine with doing that if we have strong opinions about it.

only the convertToRoute is used in another file, and seemed simple enough to duplicate it, but we can make a module for it if we want.

**Comment by ScottDowne** [comment]
fixes #1757

**Comment by ScottDowne** [comment]
@mmmavis r?

File: pages/web-literacy/WebLiteracyPage.jsx (hunk 1 above)
Comment (mmmavis) [inline pages/web-literacy/WebLiteracyPage.jsx]: maybe we can move them to `/lib/util.js` ?

**Comment by mmmavis** [comment]
Looks great to me! Left one comment https://github.com/mozilla/teach.mozilla.org/pull/1761/files#r57613278. Feel free to hit merge once we've moved the helper functions to `lib/util.js` :smile:

File: pages/web-literacy/WebLiteracyPage.jsx (hunk 1 above)
Comment (ScottDowne) [inline pages/web-literacy/WebLiteracyPage.jsx]: I might update the names too. Outside of this file the context gets lost, so I should be more clear.

But yeah, moving them to utils is fine with me.

Comment (ScottDowne) [inline pages/web-literacy/WebLiteracyPage.jsx]: Eh, keeping the names as is, should be fine.

**Comment by ScottDowne** [comment]
@mmmavis updated. I would prefer if you took another quick look.

**Comment by mmmavis** [comment]
looks great to me. merging it now! :+1:

**Pull request merged**